lib/docker_manager/upgrader.rb

```diff
@@ -111,32 +118,41 @@ def upgrade
 
     run("bundle install --deployment --without test --without development")
     percent(30)
-    run("bundle exec rake multisite:migrate")
+    run("SKIP_POST_DEPLOYMENT_MIGRATIONS=1 bundle exec rake multisite:migrate")
     percent(40)
     log("***  Bundling assets. This will take a while *** ")
     less_memory_flags = "RUBY_GC_MALLOC_LIMIT_MAX=20971520 RUBY_GC_OLDMALLOC_LIMIT_MAX=20971520 RUBY_GC_HEAP_GROWTH_MAX_SLOTS=50000 RUBY_GC_HEAP_OLDOBJECT_LIMIT_FACTOR=0.9 "
     run("#{less_memory_flags} bundle exec rake assets:precompile")
 
-    percent(100)
-    publish('status', 'complete')
-    log_version_upgrade
+    percent(80)
 
-    log("***********************************************")
-    log("*** After restart, upgrade will be complete ***")
-    log("***********************************************")
     log("Restarting unicorn pid: #{launcher_pid}")
+    Process.spawn("bundle exec ruby #{Rails.root}/plugins/docker_manager/lib/docker_manager/post_deploy.rb #{@user_id} #{@repos.map(&:path).join(",")} #{@from_version} #{unicorn_worker_pids.join(',')}")
```

Comment:
only concern here is memory, are we going to be too tight at this point to spawn this?

Reply:
also definitely do all the low memory use flags here.

Reply:
@SamSaffron Good point let me test it out on a 1 GB ram DO droplet.